src/settings.py

```diff
@@ -153,6 +153,11 @@ class GlobalSettings:
     doors_birrt_smooth_amt = 50
     doors_draw_debug = False
 
+    # coffee env parameters
+    coffee_num_cups_train = [1, 2]
+    coffee_num_cups_test = [3, 4, 5, 6]
```

Comment:
very cool that we're able to get up to 6 cups. how fast is oracle?

Reply:
yup!
```python src/main.py --env coffee --approach oracle --seed 0 --coffee_num_cups_test "[6]"```
...
```
Tasks solved: 50 / 50
Average time for successes: 0.34843 seconds
```